claim/README.md

```diff
@@ -51,7 +55,8 @@ This reloads the agent claiming state from disk.
 
 ## Netdata agent command line
 
-The user can trigger agent claiming by calling the `netdata` service binary with the additional command line parameters:
+A Space's administrator can also trigger agent claiming by calling the `netdata` service binary with the additional
+command line parameters:
```

Comment:
We should explain that this only works if netdata is it already running, I.e as an initialization or installation step.